sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 7.0.0b3 (Unreleased)
+
+**New Features**
+
+* Added `AutoComplete` as a helper, a context manager to complete a message on successful scope exit or abandon otherwise.
+
 ## 7.0.0b2 (Unreleased)
```

Comment:
How are both b2 and b3 unreleased?

Reply:
I initially didn't know if we were bringing this into b2 and was hoping to avoid a merge conflict by editing it :P  Will merge, adam mentioned the same.